pkg/model/requests_processing.go

```diff
@@ -101,18 +122,18 @@ func saturateClamp(fractionUtilised float64) float64 {
 }
 
 // returns Sakasegawa's approximation for expected queueing time for an M/M/m queue
-func sakasegawaApproximation(fractionUtilised, maxRPS float64, baseServiceTime time.Duration) time.Duration {
-	powerTerm := math.Sqrt(2*(maxRPS+1)) - 1
-	utilizationTerm := math.Pow(fractionUtilised, powerTerm) / (maxRPS * (1 - fractionUtilised))
+func sakasegawaApproximation(fractionUtilised, totalCPUCapacity float64, baseServiceTime time.Duration) time.Duration {
+	powerTerm := math.Sqrt(2*(totalCPUCapacity+1)) - 1
+	utilizationTerm := math.Pow(fractionUtilised, powerTerm) / (totalCPUCapacity * (1 - fractionUtilised))
```

Comment:
@jchesterpivotal will you take a look at this?  Is this okay?

Reply:
Not as such. Sakasegawa's approximation is for calculating slowdown based on the number of requests that are processing concurrently; it mostly represents contention between individual units of work in a single machine.

Utilization in a queue theoretic sense is about the fraction of the time a machine spends processing a unit of work. As arrival rates rise, utilization will approach 1.0 and time in queue will become exponential, proportional to `(utilization / (1 - utilization))`.

I think that the exponential slowdown formula should work instead. So we'd replace `sakasegawaApproximation` with `kingmanUtilizationApproximation`, after [Kingman's Formula](https://en.wikipedia.org/wiki/Kingman%27s_formula).

Reply:
@jchesterpivotal Please see e91267565b8ba92593c89fe969f1d65a35e99baf commit and take a look at https://docs.google.com/document/d/13FQa8eOeVQktmuHhlPgBm5n9lO7pxMvcK7PzsrL-HWU/edit# . I completely refined the model. And it seems that Kingman's Formula doesn't work in this model. Because in a current model real cpu allocation happens and Sakasegawa's approximation is what we really need.

Reply:
For example,  we have cpu_capacity = 1000 millicores and come a request with cpu_time = 100 millis. 
1) if cpu_utilization = 99%, then processing time for this request will be 10s
2) if cpu_utilization = 0%, then processing time for this request will be 1s

Reply:
Ah, OK, that makes sense and the figure seems basically sound. We can revise if necessary.